manifests/m/MasterPackager/MasterPackager/22.3.8241.0/MasterPackager.MasterPackager.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+PackageIdentifier: MasterPackager.MasterPackager
+PackageVersion: 22.3.8241.0
+PackageLocale: en-US
+Publisher: Master Packager Ltd.
+PublisherUrl: https://www.masterpackager.com/
+PackageName: Master Packager
+PackageUrl: https://www.masterpackager.com/
+License: Copyright (c) 2022 Master Packager Ltd. All rights reserved.
```

```suggestion
Copyright: Copyright (c) 2022 Master Packager Ltd. All rights reserved.
License: Proprietary
```
A copyright is not a license